modules/portal/portal-api/src/main/java/com/enonic/xp/portal/owasp/SanitizeType.java

```diff
@@ -0,0 +1,6 @@
+package com.enonic.xp.portal.owasp;
+
+public enum SanitizeType
+{
+    FUNCTION, PROCESSOR
```

Comment:
`FUNCTION` - probably should be called `STRICT`
`PROCESSOR` - probably should be called `RICH_TEXT`